trin-core/src/portalnet/types/content_keys.rs

```diff
@@ -176,18 +176,20 @@ impl StateContentKey {
                 input.append(&mut address.to_vec());
                 input.append(&mut path.to_vec());
                 input.append(&mut node_hash.to_vec());
-                Ok(keccak(&hex::encode(input))?)
+                Ok(sha256(&input))
             }
             StateContentKey::AccountTrieProofKey(AccountTrieProof { address, .. }) => {
-                Ok(keccak(&hex::encode(address.to_vec()))?)
+                Ok(keccak256(address))
```

Comment:
Is there a reason this hash function was left as `keccak256` while the others were changed to `sha256`?

Reply:
Keccak is used for AccountTrieProof and ContractStorageTrieProof because apparently it is necessary to preserve the ordering of the trie data.